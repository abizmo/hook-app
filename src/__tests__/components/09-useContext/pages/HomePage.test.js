import { mount } from 'enzyme'
import UserContext from '../../../../components/09-useContext/context/UserContext'
import HomePage from "../../../../components/09-useContext/pages/HomePage"

describe('HomePage tests', () => {
  const user = {
    name: 'Fernando',
    email: 'email@test.com'
  }
  const setUser = jest.fn()

  const wrapper = mount(
    <UserContext.Provider value={{user, setUser}}>
      <HomePage />
    </UserContext.Provider>
  )

  test('should render and match snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })
  
})
