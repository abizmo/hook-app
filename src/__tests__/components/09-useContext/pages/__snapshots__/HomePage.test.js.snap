// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomePage tests should render and match snapshot 1`] = `
<div>
  <h1>
    Home
  </h1>
  <hr />
  <pre>
    {
   "name": "Fernando",
   "email": "email@test.com"
}
  </pre>
</div>
`;
